scripts/test_tag

```diff
@@ -6,18 +6,18 @@
 # The workflow for that is to edit `.github/workflows/schemacode_ci.yml` and add a secondary branch name to the `on` section right after master.
 # This will allow you to keep pushing to the branch as if it were master and unix-time tagged archives being uploaded to test.pypi.
 
-VERSION_FILE="src/schema/SCHEMA_VERSION"
+DATETIME="$(date +%Y%m%d%H%M)"
```

Comment:
YYYYMMDDHHMM is more readable than seconds since the epoch. I don't plan to tag more than 1/min...

Reply:
Do we need this variable to mean anything though? the reason why I used unix epoch was to make sure that even if someone's being hectic there won't be any conflicts... In the end these are throwaway builds (https://test.pypi.org/project/bidsschematools/#history), we barely even care about the order much less identifying the date.... which is listed by test.pypi anyway.

Reply:
I don't think we need it to, but I think having something interpretable in the tag name and not just the commit metadata will make it easier to look through a list. That said, I'm okay reverting.